src/responder/common/cache_req/plugins/cache_req_host_by_name.c

```diff
@@ -59,6 +60,9 @@ cache_req_host_by_name_lookup(TALLOC_CTX *mem_ctx,
     *_result = result;
 
     return EOK;
+#else
+    return ERR_INTERNAL;
```

Comment:
Wouldn't be worth to add a debug message here saying that the SSSD was not built with ssh support and that's the reason we're returning ERR_INTERNAL?